docs/development/developing-dapr.md

```diff
@@ -33,8 +33,8 @@ This document helps you get started developing Dapr. If you find any problem whi
 1. The Go language environment [(instructions)](https://golang.org/doc/install#windows).
    Make sure that your GOPATH and PATH are configured correctly - You may set environment variables through the "Environment Variables" button on the "Advanced" tab of the "System" control panel. Some versions of Windows provide this control panel through the "Advanced System Settings" option inside the "System" control panel.
    ```
-   GOPATH=c:\go
-   PATH=%GOPATH%\bin;...
+   set GOPATH=c:\go
+   set PATH=%PATH%;%GOPATH%\bin
```

Comment:
I would keep the previous one because the instruction is to set GOPATH and PATH in `Advanced system settings`, not on the command window.

Reply:
Sorry....had missed reading the env variables line when I went through the setup. I had tried setting the env vars in the command window. Closed #1510 and reverted my change